src/test/java/io/confluent/examples/streams/interactivequeries/WordCountInteractiveQueriesExampleTest.java

```diff
@@ -147,12 +140,14 @@ public void shouldDemonstrateInteractiveQueries() throws Exception {
         }
     });
     kafkaStreams.start();
-  
+
+    System.out.println("Streams application service starts at " + baseUrl);
+
+    assertTrue("streams failed to start within timeout", startupLatch.await(30, TimeUnit.SECONDS));
```

Comment:
Why do we move this?

Reply:
Intention is to test streams instance before starting the rest proxy service: I guess correctness-wise there is not much difference, just feels cleaner this way.